README.md

```diff
@@ -22,25 +22,42 @@ The Primer CSS repo is managed as a monorepo that is composed of many npm packag
 
 This repository is distributed with [npm][npm]. After [installing npm][install-npm], you can install `primer-css` with this command.
 
-```
+```sh
 $ npm install --save primer-css
 ```
 
 ## Usage
 
-The source files included are written in [Sass][sass] (`scss`) You can simply point your sass `include-path` at your `node_modules` directory and import it like this.
+The source files included are written in [Sass][sass] (SCSS). You can point your Sass `include-path` at your project's `node_modules` directory and import it like this:
```

Comment:
Thanks @mdo. @shawnbot Would you be down to open a new pr? This doesn't have to hold up the new release today, but should be added.